backend/src/agent/persistence/main.go

```diff
@@ -128,4 +134,6 @@ func init() {
 	flag.StringVar(&namespace, namespaceFlagName, "", "The namespace name used for Kubernetes informers to obtain the listers.")
 	flag.Int64Var(&ttlSecondsAfterWorkflowFinish, ttlSecondsAfterWorkflowFinishFlagName, 604800 /* 7 days */, "The TTL for Argo workflow to persist after workflow finish.")
 	flag.IntVar(&numWorker, numWorkerName, 2, "Number of worker for sync job.")
+	flag.Float64Var(&clientQPS, clientQPSFlagName, 5, "The maximum QPS to the master from this client.")
```

Comment:
Would you mind adding a comment with link to default value of go-client?
I think that helps future maintainers to learn the context of how default is decided.

Reply:
Sure, added in new commit.